analyticsdataserver/settings/base.py

```diff
@@ -205,8 +205,12 @@
     'analytics_data_api.v0',
 )
 
+OPTIONAL_APPS = (
+    'enterprise_data',
+)
```

Comment:
Seems like this should be in `THIRD_PARTY_APPS`, there really isn't anything optional about it.